### DISCUSSION TITLE
non kotlin files

### DISCUSSION THREAD

**Discussion by IvanSanchez-uam** [detekt/detekt#5477] [Q&A] [answered]
Does any of you know if we can analyze non kotlin files, for instances our README file on our project or gradle files?
I have tried using the visitFile() to see if we can get the name of the gradle files (properties, settings , etc) and never saw them

**Comment by schalkms** [comment, marked as answer]
That’s out of scope for detekt. Kotlin script files (.kts) are supported, however.
